apps/bookmark/js/bookmark_editor.js

```diff
@@ -79,6 +101,7 @@ var BookmarkEditor = {
         this.installAppButtonListener = this._installApp.bind(this);
         this.installAppButton.addEventListener('click',
           this.installAppButtonListener);
+        this._renderAppIcon(manifestData, manifestURL, this.APP_ICON_SIZE);
```

Comment:
I would like to deal with the icon after showing the section and the name of the app